static/keys/private.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I'd like to see this file encrypted with the sandbox GPG key please

Reply:
This is now addressed.